portfolio.js

```diff
@@ -13,15 +15,26 @@ Project.prototype.toHtml = function() {
   $newProject.find('a').text(this.title);
   $newProject.attr('.date-created', this.dateCreated);
   $newProject.attr('src', this.image);
-  $newProject.attr('href', this.fileSource); //need to chain .html jQ method on?
+  $newProject.attr('.description', this.description);
+  // $newProject.attr('href', this.fileSource); //need to chain .html jQ method on?
   $newProject.removeClass('template');
   return $newProject;
 };
 
+portfolio.handleMainNav = function() {
```

Comment:
You'll want to keep to an MVC structure and put this functionality into another file (projectView?). This deals with DOM manipulation and thus would be part of the view :)